src/store/common/reducer.ts

```diff
@@ -0,0 +1,12 @@
+import { createReducer } from '@reduxjs/toolkit';
+import { CommonState, TOGGLE_ANNOTATION_VISIBILITY } from './types';
```

Comment:
It seems odd to have a `common` and a `mode` reducer. Should they be combined? Is `common` the name we want to use?

Reply:
> It seems odd to have a `common` and a `mode` reducer. Should they be combined? Is `common` the name we want to use?

@ConradJChan what do you think?`